common_utils/shareMem.go

```diff
@@ -0,0 +1,58 @@
+package common_utils
+
+import (
+	"fmt"
+	"syscall"
+	"unsafe"
+)
+
+var (
+	memKey = 7749
```

Comment:
>memKey [](http://example.com/codeflow?start=1&length=6)

Add comment? what is the key?

Reply:
updated

Reply:
Is it a shared credential between server and gnmi_dump?

Reply:
Yes